boot/bootchain.go

```diff
@@ -0,0 +1,173 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package boot
+
+import (
+	"bytes"
+	"encoding/json"
+	"sort"
+)
+
+type bootChain struct {
+	Model          string      `json:"model"`
+	BrandID        string      `json:"brand-id"`
+	Grade          string      `json:"grade"`
+	ModelSignKeyID string      `json:"model-sign-key-id"`
```

Comment:
Should we also include Series?

Reply:
no,  until we have a reason to (that means two different series for models)